examples/openai/autologging/README.md

```diff
@@ -0,0 +1,21 @@
+# OpenAI Autologging Examples
+
+## Using OpenAI client
+
+The recommended way of using `openai` is to instantiate a client
+using `openai.OpenAI()`. You can run the following example to use
+autologging using such client.
+
+```shell
+python3 examples/openai/autologging/instantiate_client.py --api-key="your-api-key"
```

Comment:
```suggestion
python examples/openai/autologging/instantiate_client.py --api-key="your-api-key"
```

can we remove `3` because we use `python` in other examples?

Reply:
sure!